src/Oracle/Oracle/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "PluginConfiguration": {
+    "Nodes": [],
```

Comment:
Why nodes are configured in the `config.json`? It should be read from the designation native contract.

Reply:
It's ip addresses of oracle nodes.

Reply:
But if the oracle nodes change, the ip addresses have to be modified manually?

Reply:
Yes. The change is a low frequency operation.

if necessary, I can add a `register` rpc method that carries the ip address newly added node, then the other nodes don't need to be restarted.

Reply:
Take a look to https://github.com/neo-project/neo/pull/2090